### PR TITLE
Add Notion docs

### LINKED ISSUE #5014: Create Notion Docs
# Improve documentation

Create Notion Documentation. Notion OAuth from https://github.com/supabase/gotrue/pull/326.

## Describe the problem

There is missing documentation for Notion.

## Describe the improvement

New Notion documentation.

## Additional context

Good for first time contributors.

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Add Notion docs, and reorder doc sidebar alphabetically.

## What is the current behavior?

Fix #5014

## What is the new behavior?

Add Notion docs, and reorder doc sidebar alphabetically.

## Additional context

issue #5014 and pull request https://github.com/supabase/gotrue/pull/326
Screenshot 2022-01-16 at 20-05-50 Login with Notion Supabase

### CODE REVIEW

**Comment by kiwicopple** [comment]
Nice @Isaiah-Hamilton !! Looks like we haven't even enabled it in Prod yet, so let's release this one as soon as we turn it on

File: web/sidebars.js [hunk 1]
```diff
@@ -140,16 +140,17 @@ module.exports = {
             'guides/auth/auth-magic-link',
             'guides/auth/auth-apple',
             'guides/auth/auth-azure',
-            'guides/auth/auth-bitbucket',
             'guides/auth/auth-discord',
+            'guides/auth/auth-bitbucket',
```
Comment (awalias) [inline web/sidebars.js:144]: unalphabetized? :P

**Review by awalias — APPROVED** [review]
this is a huge help @Isaiah-Hamilton , just one comment but looks great otherwise

**Comment by awalias** [comment]
also related: https://github.com/supabase/supabase/issues/5180

File: web/sidebars.js (hunk 1 above)
Comment (Isaiah-Hamilton) [reply web/sidebars.js:144]: oops don't know how that happened  😅

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**